resources/ts/Components/ContactsTableContent.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { ComponentProps } from "react";
+import { ContactsTableProps } from "./ContactsTable";
+import { classNames } from "../utils/classNames";
+import { ButtonIcon } from "./ButtonIcon";
+import { faPencil, faTrash } from "@fortawesome/free-solid-svg-icons";
+import { useTranslation } from "react-i18next";
+import { ContactFormData } from "./Integrations/Detail/ContactInfo";
+import { Contact } from "../Pages/Integrations/Index";
+type Props = { desktop?: boolean; mobile?: boolean } & ContactsTableProps &
+  ComponentProps<"div">;
+
+export const ContactsTableContent = ({
+  desktop,
+  mobile,
```

Comment:
You could also use a display prop instead with type `'mobile' | 'desktop'`
But this approach is also perfect 👍🏻